docs/General/Meetings/journal_club.md

```diff
@@ -0,0 +1,73 @@
+# Welcome to Journal Club!  
+
+Whether you’re new to the lab or looking for a refresher, this guide will help you navigate **Journal Club (JC)**—our weekly session to explore research papers, exchange ideas, and grow as scientists. Let’s dive in!  
+
+## **What Is Journal Club?**
+
+Journal Club is a weekly session designed to:
+
+- **Discuss**: Explore research papers relevant to our lab or scientific interests.
+- **Analyze**: Evaluate the strengths and weaknesses of each study.
+- **Collaborate**: Share insights on how the paper could impact or inspire our work.
+- **Apply**: Optionally consider incorporating the findings or methods into your projects.
+
+It’s all about **learning**, **critiquing**, and **getting inspired**.
+
+---
+
+## **When and Where?**
+
+- **Day**: Wednesdays  
+- **Time**: 1:00–2:00 PM  
```

Comment:
The time is 12:00 right? Should change that :)

Reply:
Thank you , it is fixed now